pydantic/_internal/_decorators.py

```diff
@@ -239,8 +325,8 @@ def build(model_dc: type[Any]) -> DecoratorInfos:  # noqa: C901 (ignore complexi
 
         # reminder: dicts are ordered and replacement does not alter the order
         res = DecoratorInfos()
-        for base in model_dc.__bases__[::-1]:
-            existing = cast(Union[DecoratorInfos, None], base.__dict__.get('__pydantic_decorators__'))
+        for base in reversed(mro(model_dc)[1:]):
+            existing: DecoratorInfos | None = base.__dict__.get('__pydantic_decorators__', None)
```

Comment:
```suggestion
            existing: DecoratorInfos | None = base.__dict__.get('__pydantic_decorators__')
```
`.get` returns `None` if not present, so I think this is equivalent. Not a big deal

Reply:
I remember that get does and pop doesn’t or the other way around, so I always try to be explicit because I get them confused, and I feel like it helps people who don’t know what the default return value is

Reply:
That is, I will gladly add four characters and some spots to alleviate the cognitive load of remembering which works, and which doesn’t. Unless they both work the same in which case I’m just dumb.